l2tdevtools/review_helpers/pylint.py

```diff
@@ -30,12 +52,22 @@ def CheckFiles(self, filenames, rcfile):
     Returns:
       bool: True if the files were linted without errors.
     """
+    version_tuple = self._GetVersion()
+
     print('Running linter on changed files.')
     failed_filenames = []
     for filename in filenames:
       print('Checking: {0:s}'.format(filename))
 
       command = 'pylint --rcfile="{0:s}" {1:s}'.format(rcfile, filename)
+      # For now disable pylint 2.1.1 and later specific checks.
+      if version_tuple >= (2, 1, 1):
```

Comment:
We still want these checks for the scaffolder, since it's py3 only. Should there be a project config for python targeted version that we consume here?